packages/starknet-snap/src/rpcs/executeTxn.test.ts

```diff
@@ -55,6 +60,33 @@ const prepareMockExecuteTxn = async (
     overallFee: BigInt(1000000000000000).toString(10),
     includeDeploy: !accountDeployed,
     unit: 'wei' as FeeTokenUnit,
+    estimateResults: [
```

Comment:
nit: 
we can make a generator to generate the estimate result, just like what i did for account & tx